pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -576,18 +576,21 @@ func (m *managerImpl) evictPod(pod *v1.Pod, resourceName v1.ResourceName, evictM
 		glog.Errorf("eviction manager: cannot evict a critical pod %s", format.Pod(pod))
 		return false
 	}
+	msg := fmt.Sprintf(message, resourceName)
 	status := v1.PodStatus{
 		Phase:   v1.PodFailed,
-		Message: fmt.Sprintf(message, resourceName),
+		Message: msg,
 		Reason:  reason,
 	}
 	// record that we are evicting the pod
 	m.recorder.Eventf(pod, v1.EventTypeWarning, reason, evictMsg)
 	gracePeriod := int64(0)
 	err := m.killPodFunc(pod, status, &gracePeriod)
 	if err != nil {
+		m.recorder.Eventf(pod, v1.EventTypeWarning, "KubeletEviction", "failed to delete pod,Err %v", err)
```

Comment:
We should limit emitting of events, from event-style guide, we shouldn’t emit events such as `failed to X`